interfaces/HO-Portal/src/app/programs/program-modal/program-modal.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
```

Comment:
Is it necessary that the `program-modal`-component is a (Angular-)'page' that needs its own `*.module.ts`-file?
It feels very verbose... :/

Reply:
I changed it to a component. I did need to add this (empty) route to programs-routing.module.ts to get it working..

```
  {
    path: '',
    component: ProgramJsonComponent,
    canActivate: [AuthGuard],
  }
```

Reply:
Update: this actually made the empty route (localhost:4200) go to ProgramJSON.. I changed this now to some random route (path: 'program-json'), so it doesn't interfere. But this is a weird solution. The point is that I shouldn't need a route for a modal, but somehow it seems to require this of me.. any ideas @elwinschmitz ?

Reply:
Ah, that explains 'bug' AB#1098 👍 

I'll see what I can look into.. But not a clear-cut-solution right away...